Profiler/install.cmd

```diff
@@ -0,0 +1,79 @@
+@echo off
+setlocal EnableDelayedExpansion
+
+rem check for admin rights
+net session >nul 2>&1
+if %errorLevel% NEQ 0 (
+	echo Failure: Execute script as Administrator.
+	pause
+	exit /B 1
+)
+
+
+set InstallSource=%CD%
+for /f "delims=" %%x in (%InstallSource%\install.cfg) do (
```

Comment:
I'm assuming that this is the part that magically reads in the install.cfg file. I'd prefer if the script prompted the user for these two questions (where to install - with a sane default directory) and whether to install the service instead of using the config file.

A config file would make sense if this needed to be automated but as this is a one-time install script I don't see the need. Asking the user makes it much more straight-forward for them to install the profiler. No documentation needed at all. Just run the script and it tells you what to do.